x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/locked_card.tsx

```diff
@@ -69,7 +69,11 @@ export const LockedPolicyCard = memo(({ title }: { title: string }) => {
               free 30-day trial, or spin up a {cloudDeploymentLink} on AWS, GCP, or Azure."
                   values={{
                     cloudDeploymentLink: (
-                      <EuiLink href="https://www.elastic.co/cloud/" target="_blank">
+                      <EuiLink
+                        href="https://www.elastic.co/cloud/"
```

Comment:
is there a way to get this URL from somewhere in kibana rather than hardcode it?